CMakeLists.txt

```diff
@@ -71,7 +72,18 @@ endif ()
 # =============================================================================
 # Check for SCALE (optional)
 # =============================================================================
-find_package(SCALE PATHS ${SCALE_DIR} COMPONENTS Shift OmnibusDriver Tpetra)
+if (Trilinos_DIR)
+  find_package(Trilinos PATHS ${Trilinos_DIR} COMPONENTS Tpetra)
+else()
+  find_package(Trilinos PATHS ${SCALE_DIR} COMPONENTS Tpetra)
+endif ()
+
+if (Trilinos_FOUND)
+  MESSAGE(STATUS "Trilinos Found!")
+  MESSAGE(STATUS "Trilinos_DIR = ${Trilinos_DIR}")
+endif ()
+
+find_package(SCALE PATHS ${SCALE_DIR} COMPONENTS nemesis ampx geometria omnibus shift)
```

Comment:
This returns FALSE without SCALE modifications.  We may need to do something different here so we don't have to modify SCALE.